utils/debug/Debug.js

```diff
@@ -18,21 +18,38 @@ function Debug(scene) {
 
     scene.viewerDiv.appendChild(chartDiv);
 
+    const leftChart = document.createElement('div');
```

Comment:
Looks nice indeed, and will definitely be useful.

One more suggestion (if it's easy to implement): I'd love to have both `visible` and `displayed` properties on the same graph :)